source/speechDictHandler/__init__.py

```diff
@@ -115,9 +114,27 @@ def sub(self, text):
 				del self[index]
 		return text
 
+
+# useful for #14689 (see below)
+IGNORE_TRAILING_WHITESPACE_LENGTH = 100

```

Comment:
Can this remain private - either scoped in `processText` or prefixed with an underscore.

```suggestion
_IGNORE_TRAILING_WHITESPACE_LENGTH = 100
```